### PR TITLE
Feature/dis 650

### PR DESCRIPTION
**Pull request by jeffbum**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      ? `default:'${cleanUpColumnDefault}'${nullable ? ', ' : ''}`
+      : '';
+    const note = col.column_comment ? `note:'${col.column_comment}'` : '';
+    const squareBrackets =
+      nullable || columnDefault || note ? `[${nullable}${columnDefault}${note}]` : '';
+    const DBMLDefiniton = `\t${col.column_name} ${dataType}${characterMaxLength}${squareBrackets}`;
+    return DBMLDefiniton;
+  });
+  convertToDBML.unshift(`Table ${tableName} {`);
+  convertToDBML.push('}\n\n');
```
Comment (korymarks) [inline index.js]: If you want to allow windows users to use this package, then you will want import `const { EOL } = require('os');` and then where \n use EOL

File: index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  const returnValue = convertToDBML.join('\n');
+  o && console.log(`creating/adding to: ${schemaName}.dbml to your output path with the dbml definition of table ${tableName}.`)
+  o ? fs.appendFile(fileName, returnValue, 'utf8', () => {}) : console.log(returnValue);
+}
+
+function tableColumnInfoQueryFunc(tableNameArr, schemaName, fileName) {
```
Comment (korymarks) [inline index.js]: function names should be actions

File: index.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  tableNameArr.forEach(tableName => {
+    const query = tableColumnInfoQuery(tableName);
+    dbClient.query(query, (err, res) => {
+      if (err) process.exit(`${tableName}: ${err}`);
+      writeToDBML(res.rows, tableName, schemaName, fileName);
+    });
+  });
+}
+
+function tablesInSchemaFunc(schemaNameArr) {
```
Comment (korymarks) [inline index.js]: function names should be actions

File: index.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+      const fileName = `${o || './'}${schemaName}.dbml`
+      if(fs.existsSync(fileName, fs.constants.R_OK | fs.constants.W_OK)) {
+        fs.writeFileSync(fileName, '', ()=> {});
+      }
+      tableColumnInfoQueryFunc(tableNameArr, schemaName, fileName);
+    });
+  });
+}
+
+function schemaQueryFunc() {
```
Comment (korymarks) [inline index.js]: this is a noun not a verb

File: index.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    .default('o', './')
+    .alias('c', 'connection_string')
+    .nargs('c', 1)
+    .describe('c', 'database connection string for the db you want to output dbml file(s).')
+    .alias('db', 'db_name')
+    .nargs('db', 1)
+    .describe('db', 'database name you want to create dbml file(s) from.')
+    .alias('t', 'timeout')
+    .nargs('t', 1)
+    .describe('t', 'how long you want process to run before it exits process.')
```
Comment (korymarks) [inline index.js]: maybe add the unit of measurement  `how long in milliseconds`...

**Review by korymarks — APPROVED** [review]
Just a few suggestions you can make and merge. I won't need to see it again. Great work Jeff!

**Comment by dbirks** [comment]
Ok added the `NODE_AUTH_TOKEN` as a secret too. I _think_ that's all it should need.

File: index.js (hunk 1 above)
Comment (jeffbum) [reply index.js]: Thanks for this. I would have never thought of this.

**Pull request merged**